src/index.ts

```diff
@@ -63,6 +64,20 @@ function updateNode () {
 	}
 }
 
+function logManualUpdateMessage () {
+	console.log(`
+@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+@     WARNING: PROBE FIRMWARE OUTDATED, PLEASE UPDATE     @
+@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+Current probe firmware is out of date and we couldn't update it automatically.
```

Comment:
Wouldn't this error also show to VMs with 512MB of RAM? It would be very confusing to them if we asked them to update a non-existent firmware

Reply:
yes let's customize for docker/hw based on the existing variables

Reply:
So if RAM is limited for virtual probe we ask to increase it or update manually, right?
And for hw probe we don't log anything as nothing could be done?

Reply:
For HW update FW, for software update the container. Just detect which case it is and print a customized message.